lighthouse-core/util-commonjs.js

```diff
@@ -111,7 +111,8 @@ class Util {
     /** @type {Map<string, Array<LH.ReportResult.AuditRef>>} */
     const relevantAuditToMetricsMap = new Map();
 
-    // Old LHR versions (<=8.6.0) do not have the "hidden" group.
+    // Old LHR versions (<9.0.0-alpha.0) do not have the "hidden" group.
```

Comment:
```suggestion
    // Old LHR versions (<9.0.0) do not have the "hidden" group.
```

I think this would be fine :)